pyro/infer/autoguide/guides.py

```diff
@@ -588,20 +588,22 @@ def get_posterior(self, *args, **kwargs):
         Returns a LowRankMultivariateNormal posterior distribution.
         """
         loc = pyro.param("{}_loc".format(self.prefix), self._init_loc)
+        scale = pyro.param("{}_scale".format(self.prefix),
+                           lambda: loc.new_full((self.latent_dim,), self._init_scale),
+                           constraint=constraints.positive)
         factor = pyro.param("{}_cov_factor".format(self.prefix),
                             lambda: loc.new_empty(self.latent_dim, self.rank).normal_(
-                                0, self._init_scale * (0.5 / self.rank) ** 0.5))
-        diagonal = pyro.param("{}_cov_diag".format(self.prefix),
-                              lambda: loc.new_full((self.latent_dim,), 0.5 * self._init_scale ** 2),
-                              constraint=constraints.positive)
+                                0, self._init_scale / self.rank ** 0.5))
+        factor = factor * scale.unsqueeze(-1)
+        diagonal = scale * scale
         return dist.LowRankMultivariateNormal(loc, factor, diagonal)
 
     def _loc_scale(self, *args, **kwargs):
         loc = pyro.param("{}_loc".format(self.prefix))
+        scale = pyro.param("{}_scale".format(self.prefix))
         factor = pyro.param("{}_cov_factor".format(self.prefix))
-        diagonal = pyro.param("{}_cov_diag".format(self.prefix))
-        scale = (factor.pow(2).sum(-1) + diagonal).sqrt()
-        return loc, scale
+        scale = scale * (factor.pow(2).sum(-1) + 1).sqrt()
+        return loc, scale * 2
```

Comment:
why 2?

Reply:
oops, fixed. The 2 was a mutation test: I added temporarily just to ensure tests were strong enough to catch that error. Then I forgot to revert the mutation.